src/Microsoft.AspNetCore.Authentication/project.json

```diff
@@ -10,6 +10,7 @@
     "keyFile": "../../tools/Key.snk"
   },
   "dependencies": {
+    "Microsoft.AspNetCore.Antiforgery": "1.0.0-*",
```

Comment:
`.Authentication` is the only package here that has a `AddXyz(...)` so it needs to be a dependency at this level